editor/components/autocompleters/block.js

```diff
@@ -24,13 +34,7 @@ export default {
 	className: 'editor-autocompleters__block',
 	triggerPrefix: '/',
 	options: once( function options() {
-		return Promise.resolve(
-			// Prioritize common category in block type options
-			sortBy(
-				getBlockTypes(),
-				( { category } ) => 'common' !== category
-			)
-		);
+		return Promise.resolve( flow( filterBlockTypes, sortBlockTypes )( getBlockTypes() ) );
```

Comment:
This is a somewhat temporary fix because @brandonpayton is working on using `getInserterItems` here which would also fix this bug as well as a lot of similar bugs.